default/stringtables/de.txt

```diff
@@ -1115,7 +1115,7 @@ OPTIONS_DB_QUICKSTART
 Beginnt ein neues Schnellstartspiel. Das Hauptmenü wird umgangen.
 
 OPTIONS_DB_CHECKED_GL_VERSION
-Speichert ob die OpenGL Version des Systems überprüft wurde. Wenn nicht, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
+Speichert ob die OpenGL Version des Systems nicht überprüft wurde. Wenn falsche, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
```

Comment:
Please remove the German translation. The old version has already been incomplete (the entire part "...after which, this option will be set to true." is missing), and the rest isn't particularly good.

I don't have the time/am not in the mood to provide a reasonably well worded translation right now, let's leave that to the next volunteer who wants to take on the German translation. 😉